gems/sorbet-runtime/lib/types/props/decorator.rb

```diff
@@ -293,7 +293,7 @@ def prop_validate_definition!(name, cls, rules, type)
       cls: PropTypeOrClass,
       rules: Rules,
     )
-    .void
+    .returns(T.anything)
```

Comment:
I assume this means that this method always just returned a truthy value before. Do we know what was broken before because of this?

Reply:
Rather: this method was `.checked(:never)`, so the underlying value it returned was being propagated through, instead of being replaced with the void singleton.